pypeit/spectrographs/apf_levy.py

```diff
@@ -372,6 +372,12 @@ def config_specific_par(self, scifile, inp_par=None):
 
         if binning == "2,2":
             par['calibrations']['slitedges']['min_edge_side_sep'] = 2.0
+            par['calibrations']['slitedges']['pad'] = 2
+            par['reduce']['skysub']['no_local_sky'] = True
```

Comment:
More out of curiosity than critique, why turn off local sky subtraction for 2x2 binning?